src/PKSim.Infrastructure/Serialization/Xml/Serializers/PopulationXmlSerializer.cs

```diff
@@ -15,20 +15,10 @@ public override void PerformMapping()
 
    public class RandomPopulationXmlSerializer : PopulationXmlSerializer<RandomPopulation>
    {
-      public override void PerformMapping()
-      {
-         base.PerformMapping();
-         Map(x => x.Settings);
```

Comment:
This is not required! I just realized it was saved twice all that time . It is saved as a separate column in the DB (see mapping)